pkg/server/image_pull.go

```diff
@@ -66,6 +77,16 @@ func (c *criContainerdService) PullImage(ctx context.Context, r *runtime.PullIma
 	return &runtime.PullImageResponse{ImageRef: digest}, err
 }
 
+// normalizeImageRef normalizes the image reference following the docker convention. This is added
+// mainly for backward compatibility.
```

Comment:
yeah but if they change it we're letting them break customers.. sort of like when they changed their version tag to non sem-ver..